<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Olympic Tournament</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(8, 26, 62);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="userPage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QStackedWidget" name="userStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="browserPage">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="browseLabel">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Parcourir</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="optionsPushButton">
                <property name="font">
                 <font>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
border: 2px solid rgb(8, 26, 62);
border-radius: 7px;</string>
                </property>
                <property name="text">
                 <string>Options</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="searchLineEdit">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                </property>
                <property name="placeholderText">
                 <string>Nom d'un tournoi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="searchPushButton">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                </property>
                <property name="text">
                 <string>Chercher</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="sortComboBox">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                </property>
                <property name="placeholderText">
                 <string>-- Trier par --</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="browserListWidget">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="movement">
               <enum>QListView::Static</enum>
              </property>
              <item>
               <property name="text">
                <string>Tournoi 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tournoi 2</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="viewPage">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="viewLabel">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Visualisation</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="backPushButton">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                </property>
                <property name="text">
                 <string>&lt;- Retour</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="tournamentNameLabel">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Nom du tournoi</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="tounamentInformationsScrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>744</width>
                 <height>431</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="playersLabel">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Participants</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QListWidget" name="playersListWidget">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>Equipe 1</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Equipe 2</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="detailsGroupBox">
                    <property name="title">
                     <string>Détail de l'équipe</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <item>
                      <widget class="QLabel" name="detailLabel">
                       <property name="font">
                        <font>
                         <pointsize>11</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTextEdit" name="detailsTextEdit">
                       <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="treeLabel">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Arbre du tournoi</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="hidePushButton">
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                    </property>
                    <property name="text">
                     <string>Cacher</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="treeImageLabel">
                  <property name="text">
                   <string>Ceci est l'image d'un arbre</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settingsPage"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="adminTitleLabel">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Administrateur</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="addAccountGroupBox">
            <property name="styleSheet">
             <string notr="true">border: 2px solid white;
border-radius: 7px;</string>
            </property>
            <property name="title">
             <string>Ajouter</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="nameAddLineEdit">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;
margin-top: 12px;</string>
               </property>
               <property name="placeholderText">
                <string>Nom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mailAddLineEdit">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
               </property>
               <property name="placeholderText">
                <string>Mail</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordAddLineEdit">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
               </property>
               <property name="placeholderText">
                <string>Mot de passe</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="roleAddComboBox">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
               </property>
               <property name="placeholderText">
                <string>--Choisir un rôle--</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="verificationAddCheckBox">
               <property name="styleSheet">
                <string notr="true">border: 0px solid white;</string>
               </property>
               <property name="text">
                <string>Je suis sûr de vouloir créer ce compte</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="addPushButton">
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 135, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                 </property>
                 <property name="text">
                  <string>Ajouter</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="deleteAccountGroupBox">
            <property name="styleSheet">
             <string notr="true">border: 2px solid white;
border-radius: 7px;</string>
            </property>
            <property name="title">
             <string>Supprimer</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QComboBox" name="roleDelComboBox">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;
margin-top: 12px;</string>
               </property>
               <property name="placeholderText">
                <string>--Choisir un rôle--</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="accountDelComboBox">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
               </property>
               <property name="placeholderText">
                <string>--Choisir le compte--</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="verificationDelCheckBox">
               <property name="styleSheet">
                <string notr="true">border: 0px solid white;</string>
               </property>
               <property name="text">
                <string>Je suis sûr de vouloir supprimer ce compte</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="delPushButton">
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(193, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
                 </property>
                 <property name="text">
                  <string>Supprimer</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>55</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modoPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playersListWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>detailLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
